### PR TITLE
Consider the free space ellipsoid

### PR DESCRIPTION
**Pull request by S-Dafarra**
It opens a port from in which it is possible to communicate the free space ellipsoid.

It requires https://github.com/robotology/unicycle-footstep-planner/pull/40

### CODE REVIEW

**Comment by S-Dafarra** [comment]
Friendly ping @GiulioRomualdi

**Comment by S-Dafarra** [comment]
Friendly ping @GiulioRomualdi

File: src/TrajectoryPlanner/src/FreeSpaceEllipseManager.cpp [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+            {
+                double a = 10, b = 10, theta = 0, centerX = 0, centerY = 0;
+
+                std::vector<std::tuple<size_t, double&> >inputs = {{0, a},
+                                                                   {1, b},
+                                                                   {2, theta},
+                                                                   {3, centerX},
+                                                                   {4, centerY}};
+
+                for (auto i : inputs)
```
Comment (GiulioRomualdi) [inline src/TrajectoryPlanner/src/FreeSpaceEllipseManager.cpp:41]:
```suggestion
                for (const auto& i : inputs)
```

**Review by GiulioRomualdi — APPROVED** [review]

**Comment by S-Dafarra** [comment]
@GiulioRomualdi can you confirm I can merge?

**Comment by GiulioRomualdi** [comment]
Yes you can merge!

**Comment by S-Dafarra** [comment]
Thanks @GiulioRomualdi!

**Pull request merged**